### DISCUSSION TITLE
Priority of new features in the next version

### DISCUSSION THREAD

**Discussion by LoveBodhi** [Wohlstand/TheXTech#414] [Polls]
We should add some new features in the next version.

The most polls stand for the feature should be done at the first.

**Comment by ds-sloth** [comment]
We're definitely interested to hear which features people want to see us implement, but there are a lot of things the developers need to consider that might not be obvious to users.

I want to make it clear that we respect the ideas everyone shares here, but we cannot promise to implement features in any particular order.
